tests/e2e/utils/track-event.js

```diff
@@ -15,11 +15,12 @@
  */
 export function trackGtagEvent( eventName ) {
 	const eventPath = '/pagead';
+	const option = { timeout: 15000 };
```

Comment:
According to the documentation the default timeout for waitForRequest is 30 seconds, but it can be overwritten with `page.setDefaultTimeout`. When originally implementing I tested it with the 30 second timeout which seemed long enough.
Ever since PR #1627 the default timeout was set at 3 seconds (bumped up to 10 seconds in this PR). Shouldn't we just set it back to the default of 30 seconds here? Or set the default timeout to 30 seconds so we only need to set it once, I don't think it's a big issue that it takes longer to timeout if it will be more stable.

Reply:
Thanks for the suggestion, adjusted it in two commits: https://github.com/woocommerce/google-listings-and-ads/pull/1636/commits/25944aed3404b8692fcd64d2bc662bbf14fb6483 and https://github.com/woocommerce/google-listings-and-ads/pull/1636/commits/9c3d297f48eca3dd9e7305743f06edefcc035d99